### ISSUE TITLE
clean slate

### ISSUE THREAD

**Issue by joshuago78** [code4lib/2017.code4lib.org#1]
Clean out data, links, and other references to the 2016 event and give ourselves a clean slate to work with.

**Comment by queryluke** [comment]
I'll take this one

**Comment by queryluke** [comment]
Clean slate is done. I did quite a bit of refactoring. A couple of things to note:
1. Working towards pretty urls... so sub pages are in sub directories /index.html. This is the [standard way to do it in jekyll](https://jekyllrb.com/docs/pages/). It also keeps the rootdir a bit cleaner
2. moved css/js/imgs to an assets dir
3. I've added a few more yml files in the _data dir. This should improve some workflows, e.g. to add a sponsor, you only need to add them to the sponsors.yml file and they'll appear on the sponsors page.
4. Finally I've created a conf.yml which is a new idea. you'll see a toggles object in there. In theory, I'm thinking certain toggle-able elements can be controlled by this yml file. For example, the registration link may appear on the main page, the attend page, and other places. By using this global option we can turn on/off the registration link everywhere it's used.  Something we can experiment with this year.

I'm still working on re-doing the readme to give an overview of how I managed all the data imports. Hope to have that done later this week.
